src/components/ThoughtAnnotation.js

```diff
@@ -31,6 +31,14 @@ import StaticSuperscript from './StaticSuperscript'
 import ContextBreadcrumbs from './ContextBreadcrumbs'
 import UrlIcon from './icons/UrlIcon'
 
+const getSubThoughtTextMarkup = (isEditing, subthought, thoughtMeta) => ({
```

Comment:
I reeeeeeeally would like you to get in the habit of adding JSDOC comments to EEEEEEEEEVERY function declaration 😊😊😊